app/controllers/IsPublicServicePensionsRemedyController.scala

```diff
@@ -52,16 +51,20 @@ class IsPublicServicePensionsRemedyController @Inject()(override val messagesApi
     with I18nSupport
     with NunjucksSupport {
 
-  private def form(memberName: String)(implicit messages: Messages): Form[Boolean] =
-    formProvider(messages("isPublicServicePensionsRemedy.error.required", memberName))
+  private def form(memberName: String, optIndex: Option[Index])(implicit messages: Messages): Form[Boolean] = {
+    optIndex match {
```

Comment:
I think it might be simpler to just pass the message key into the form from the controller rather than passing in the option index